docs/developer-docs/gas-cost.md

```diff
@@ -70,12 +68,10 @@ To derive the estimated cost for a GB Storage per month, we assume a 30 day mont
 | Ingress Byte Reception               | For every byte received in an ingress message                                                                    | $0.00000000261772           | $0.00000000684534           |
 | GB Storage Per Second                | For storing a GB of data per second                                                                              | $0.00000016622522           | $0.00000043474178           |
 |                                      |                                                                                                                  |                             |                             |
-| *Chain-key signatures*               |                                                                                                                  |                             |                             |
-| Threshold ECDSA signing              | For computing one threshold ECDSA signature (`sign_with_ecdsa`)                                                  | $0.0130886                  | $0.0342317                  |
-|                                      |                                                                                                                  |                             |                             |
 | *HTTPS outcalls*                     |                                                                                                                  |                             |                             |
-| HTTPS outcall request                | For sending an HTTPS outcall to a server outside the IC, per message (`http_request`)                            | $0.0005235440               | $0.0013692689               |
-| HTTPS outcall payload                | For sending an HTTPS outcall to a server outside the IC, per request and reserved response byte (`http_request`) | $0.0000001308860            | $0.0000003423166            |
+| HTTPS outcall request                | For sending an HTTPS outcall to a server outside the IC, per message (`http_request`)                            | $0.0000643173804               | 0.0002242862496
+| HTTPS outcall request payload (per byte) | For sending an HTTPS outcall to a server outside the IC, per request byte (`http_request`)                   | $0.000000006806072            | $0.000000017800496
```

Comment:
`payload` may not be the best word here. Maybe `size` is better. Because people may think you pay only for HTTP payload bytes, while the payment is for the whole message size including URL, headers, etc.

Reply:
OK, changed it.